---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid type nature for generic argument. REAL is no String.", range: [SourceRange { range: 432..440 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. UDINT is no String.", range: [SourceRange { range: 442..454 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DINT is no String.", range: [SourceRange { range: 456..466 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. TIME is no String.", range: [SourceRange { range: 468..476 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. BYTE is no String.", range: [SourceRange { range: 478..486 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. CHAR is no String.", range: [SourceRange { range: 497..505 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DATE is no String.", range: [SourceRange { range: 507..515 }], err_no: type__invalid_nature }

